tools/Nethermind.Tools.Kute/Config.cs

```diff
@@ -77,4 +77,46 @@ public static class Config
     {
         Description = "If true then each batched request will be unwraped to single requests"
     };
+
+    public static Option<IEnumerable<(string, string)>> Tags { get; } = new("--tags", "-t")
```

Comment:
We should update the README with examples on how to use `--tags` and prometheus options.